src/global_distrib/mod_global_distrib_utils.erl

```diff
@@ -155,18 +157,22 @@ create_ets(Name, Type) ->
 -spec resolve_endpoints([{inet:ip_address() | string(), inet:port_number()}]) ->
                                [endpoint()].
 resolve_endpoints(Endpoints) ->
-    lists:flatmap(
-      fun({Addr, Port}) ->
-              case to_ip_tuples(Addr) of
-                  {ok, IpAddrs} ->
-                      [{IpAddr, Port} || IpAddr <- IpAddrs];
-                  {error, {Reasonv6, Reasonv4}} ->
-                      ?ERROR_MSG("Cannot convert ~p to IP address: IPv6: ~s. IPv4: ~s.",
-                                 [Addr, inet:format_error(Reasonv6), inet:format_error(Reasonv4)]),
-                      error({Reasonv6, Reasonv4})
-              end
-      end,
-      Endpoints).
+    lists:foldl(fun resolve_endpoint/2, [], Endpoints).
+
+resolve_endpoint({Addr, _Port} = E, Endpoints) when is_tuple(Addr) ->
+    [E | Endpoints];
+resolve_endpoint({Addr, Port}, Endpoints) ->
+    case to_ip_tuples(Addr) of
+        {ok, IpAddrs} ->
+            Resolved = Endpoints ++ [{IpAddr, Port} || IpAddr <- IpAddrs],
+            ?INFO_MSG_IF(is_domain(Addr), "Domain ~p resolved to: ~p", [Addr, IpAddrs]),
```

Comment:
Will this message ever trigger? `Addr` cannot be `{domain, _}` because it's used as an argument for `to_ip_tuples/1` which takes a string or an IP tuple.

Reply:
`Addr` doesn't have to be of the form `{domain, _}` for `is_domain/1` to return true.